front_end/panels/ai_chat/core/AgentNodes.ts

```
@@ -4,17 +4,20 @@
 
 import type { getTools } from '../tools/Tools.js';
```

ESLint

[error] `../tools/Tools.js` type import should occur after type import of `../LLM/LLMTypes.js`

(import/order)